<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_Limit</class>
 <widget class="QWidget" name="widget_Limit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Limit</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Max likes per day:</string>
           </property>
           <property name="buddy">
            <cstring>sb_like</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Max unlikes per day:</string>
           </property>
           <property name="buddy">
            <cstring>sb_unlike</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Max follows per day:</string>
           </property>
           <property name="buddy">
            <cstring>sb_follow</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Max unfollows per day:</string>
           </property>
           <property name="buddy">
            <cstring>sb_unfollow</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Max comments per day:</string>
           </property>
           <property name="buddy">
            <cstring>sb_comment</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Max likes to like:</string>
           </property>
           <property name="buddy">
            <cstring>sb_liketolike</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Max followers to follow:</string>
           </property>
           <property name="buddy">
            <cstring>sb_foltofol</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Min followers to follow:</string>
           </property>
           <property name="buddy">
            <cstring>sb_min_foltofol</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Max following to follow:</string>
           </property>
           <property name="buddy">
            <cstring>sb_fingtofol</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QSpinBox" name="sb_like">
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly</set>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::UpDownArrows</enum>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>2000</number>
           </property>
           <property name="value">
            <number>999</number>
           </property>
           <property name="displayIntegerBase">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sb_unlike">
           <property name="maximum">
            <number>2000</number>
           </property>
           <property name="value">
            <number>50</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sb_follow">
           <property name="maximum">
            <number>2000</number>
           </property>
           <property name="value">
            <number>999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sb_unfollow">
           <property name="maximum">
            <number>2000</number>
           </property>
           <property name="value">
            <number>999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sb_comment">
           <property name="maximum">
            <number>500</number>
           </property>
           <property name="value">
            <number>149</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sb_liketolike">
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="value">
            <number>1500</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sb_foltofol">
           <property name="maximum">
            <number>100000</number>
           </property>
           <property name="value">
            <number>10000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sb_min_foltofol">
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sb_fingtofol">
           <property name="maximum">
            <number>8000</number>
           </property>
           <property name="value">
            <number>7500</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Min following to follow:</string>
           </property>
           <property name="buddy">
            <cstring>sb_min_fingtofol</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Max followers to following_ratio:</string>
           </property>
           <property name="buddy">
            <cstring>sb_fertofing</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Max following to followers_ratio:</string>
           </property>
           <property name="buddy">
            <cstring>sb_fingtofer</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Min media_count to follow:</string>
           </property>
           <property name="buddy">
            <cstring>sb_min_media</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Like delay:</string>
           </property>
           <property name="buddy">
            <cstring>sb_like_delay</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Unlike delay:</string>
           </property>
           <property name="buddy">
            <cstring>sb_unlike_delay</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Follow delay:</string>
           </property>
           <property name="buddy">
            <cstring>sb_follow_delay</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Unfollow delay:</string>
           </property>
           <property name="buddy">
            <cstring>sb_unfollow_delay</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Comment delay:</string>
           </property>
           <property name="buddy">
            <cstring>sb_comment_delay</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QSpinBox" name="sb_min_fingtofol">
           <property name="maximum">
            <number>8000</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sb_fertofing">
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="value">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sb_fingtofer">
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sb_min_media">
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sb_like_delay">
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>40</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sb_unlike_delay">
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>40</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sb_follow_delay">
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>40</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sb_unfollow_delay">
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>30</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sb_comment_delay">
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>400</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Proxy:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_proxy">
       <property name="statusTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="accessibleName">
        <string/>
       </property>
       <property name="accessibleDescription">
        <string/>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="placeholderText">
        <string>Ip Adress</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_save">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
